Functions/xASL_bids_CreateDatasetDescriptionTemplate.m

```diff
@@ -77,14 +78,19 @@
 		
     % Optional fields
     for iCell = 1:numel(bidsPar.datasetDescription.Optional)
-		% Only copy if defined
-		if isfield(draft,bidsPar.datasetDescription.Optional{1,iCell})
-			json.(bidsPar.datasetDescription.Optional{1,iCell}) = draft.(bidsPar.datasetDescription.Optional{1,iCell});
-		end
+        % Only copy if defined
+        if isfield(draft,bidsPar.datasetDescription.Optional{1,iCell})
+            json.(bidsPar.datasetDescription.Optional{1,iCell}) = draft.(bidsPar.datasetDescription.Optional{1,iCell});
+        end
+        % Add version which was used for the import to the Acknowledge field (Imported with xASL v1.x.x)
+        if strcmp(bidsPar.datasetDescription.Optional{1,iCell},'Acknowledgements')
+            json.Acknowledgements = ['Imported with ExploreASL ' versionExploreASL];
```

Comment:
lets end this statement with a fullstop - see my comment later.

Reply:
773a8c2 👍